src/common/components/select-repository-list/index.js

```diff
@@ -130,18 +130,29 @@ export class SelectRepositoryListComponent extends Component {
   renderRepoAmount() {
     const { ids, isFetching } = this.props.repositories;
     const { selectedRepositories } = this.props;
+    const { filteredRepos } = this.props;
+    let filteredAmount = null;
 
     // Return nothing until isFetching completes
     if (isFetching || ids.length === 0) {
       return;
     }
 
+    if (filteredRepos.length < ids.length && filteredRepos.length > 1) {
+      filteredAmount = <span>{ filteredRepos.length } matches in </span>;
```

Comment:
We don't need to do `<span>` here. There is no need to complicate it. `filteredAmount` can be simply a string that we will put into `<strong>` when returning from this function.